iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/json/ResourceDeclarationImplTest.java

```diff
@@ -90,7 +90,6 @@ void shouldParseResource() {
   @ParameterizedTest
   @ValueSource(strings = {
     "\"type\": 5,                            \"apiVersion\": \"2022-12-29\", \"name\": \"myResource\"",
-    "\"type\": \"Microsoft.Kusto/clusters\", \"apiVersion\": 5,              \"name\": \"myResource\"",
```

Comment:
This is an invalid value before parsing, from the perspective of JSON. But after parsing and building the tree we can have an arbitrary Expression here